packages/app/config/jest/setupTests.js

```diff
@@ -26,4 +29,13 @@ jest.mock("workers", () => {
   return {
     getOpScriptWorker: () => {}
   };
-});
\ No newline at end of file
+});
+
+jest.setTimeout(15000)
+configure({
+  asyncUtilTimeout: 15000
+})
```

Comment:
The problem seemed to be that in the CI, the tests tended to timeout - so I increased the timeout time from 5s to 15s.

Reply:
@jgehrcke @MoSattler we're still seeing this problem https://buildkite.com/opstrace/scheduled-main-builds/builds/2532#cd42b3ef-4d78-4759-87cd-75c5bba9261e/505-1137 and https://buildkite.com/opstrace/prs/builds/4793#1637cf31-dd2b-4e04-904a-b458ececb5f3/517-627

going to try to bump this timeout an order of magnitude to see if that helps

Reply:
https://github.com/opstrace/opstrace/pull/926